app/models/order.rb

```diff
@@ -1,9 +1,20 @@
 class Order < ApplicationRecord
+  validates :total_price, presence: true
+
   belongs_to :user
 
   has_many :order_items
   has_many :items, through: :order_items
 
   enum status: ['ordered', 'paid', 'canceled', 'completed']
 
+  def self.get_by_status(find_status)
+    orders = find_by(status: find_status)
+     if orders == nil
```

Comment:
this conditional is to deal with situations in which there are no orders of a particular status